apps/chat/src/utils/auth/auth-callbacks.ts

```diff
@@ -12,9 +12,19 @@ import { TokenSet } from 'openid-client';
 
 const waitRefreshTokenTimeout = 5;
 
+const safeDecodeJwt = (accessToken: string) => {
```

Comment:
It might be better to move this function to the jwt.ts file inside utils/server.
What do you think about it?